common/interfaces/user_interface/ui_joystick_training.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file    ui_joystick_training.h
+ * @author  Cypherock X1 Team
+ * @brief   UI component to train user for joystick.
+ * @copyright Copyright (c) 2023 HODL TECH PTE LTD
+ * <br/> You may obtain a copy of license at <a href="https://mitcc.org/"
+ * target=_blank>https://mitcc.org/</a>
+ *
+ */
+#ifndef UI_JOYSTICK_TRAINING_H
+#define UI_JOYSTICK_TRAINING_H
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+
+#include "ui_common.h"
+
+/*****************************************************************************
+ * MACROS AND DEFINES
+ *****************************************************************************/
+
+/*****************************************************************************
+ * TYPEDEFS
+ *****************************************************************************/
+
+typedef enum joystick_actions {
+  JS_ACTION_UP = LV_KEY_UP,
+  JS_ACTION_DOWN = LV_KEY_DOWN,
+  JS_ACTION_LEFT = LV_KEY_LEFT,
+  JS_ACTION_RIGHT = LV_KEY_RIGHT,
+  JS_ACTION_CENTER = LV_KEY_ENTER,
```

Comment:
ATCG: dont' use acronyms. Also there's a format to follow for enums, if you wanna comply completely.